LabTools/CoreWidgets/ScriptWidget.py

```diff
@@ -82,14 +148,17 @@ def add_widget_into_main(parent):
 
     fname = IOTool.get_script_name(config_file_path=parent.config_file)
 
-    parent.widgets['ScriptWidget'].scriptFileLineEdit.setText(fname)
+    #parent.widgets['ScriptWidget'].scriptFileLineEdit.setText(fname)
```

Comment:
please do not push code which is commented out :)
Possible solution for that now:
- checkout your branch: `git checkout feature/dropdown_script_selector`
- erase the line locally and stage the change with `git add LabTools/CoreWidgets/ScriptWidgets.py`
- melt this change in the previous commit: `git commit --amend`
- force push your changes on the remote (only do that when the other people working on the branch are aware of it, i.e. tell them beforehand) : `git push -f`